index.html

```diff
@@ -1,202 +1,503 @@
-		/>
-		<link rel="stylesheet" href="css/normalize.css" />
-		<link rel="stylesheet" href="css/global.css" />
-		<!-- In a new branch, rename the page.css file adding the name of that actual page (Example: home.css) -->
-		<link rel="stylesheet" href="css/page.css" />
-	</head>
-
-	<body>
-		<!-- Header Navigation-->
-
-		<!-- Hero -->
-		<h1>If These Lands Could Talk</h1>
-
-		<!-- Main content -->
-
-		<!-- Footer -->
-
-		<footer>
-			<div class="wrapper">
-				<div class="footer-main">
-					<div class="footer-logo-btn">
-						<!-- logo -->
-						<a href="./index.html">
-							<img
-								src="./assets/icons/global/itlct-logo.svg"
-								alt="If These Lands Could Talk Logo"
-							/>
-						</a>
 
-						<!-- Button for modal -->
-						<button class="btn-la">Land Acknowledgement</button>
-					</div>
-					<!-- Links list -->
-					<!-- need to replace with the real page links -->
-					<ul>
-						<li>
-							<a href="/index.html">Home</a>
-						</li>
-						<li>
-							<!-- Need to double check once Our Impact team is started -->
-							<a href="/impact.html">Our Impact</a>
-						</li>
-						<li>
-							<a href="/values.html">Our Values</a>
-						</li>
-						<li>
-							<a href="/events.html">Events & Programs</a>
-						</li>
-						<li>
-							<a href="/team.html">Our Team</a>
-						</li>
-						<li>
-							<a href="#">Join the Team</a>
-						</li>
-						<li>
-							<a href="/contact.html">Contact</a>
-						</li>
-						<li>
-							<a href="#">Support Us</a>
-						</li>
-					</ul>
-
-					<!-- form -->
-					<form action="" class="">
-						<h2 class="h4-styles">Stay up to date with us</h2>
-						<p>
-							Register with our newsletter to receive updates on our website,
-							events, and more!
-						</p>
-						<div class="text-input">
-							<label class="hidden" for="newsletter-input">Newsletter</label>
-							<input
-								type="email"
-								name="newsletter-input"
-								id="newsletter-input"
-								size="40"
-								placeholder="Enter your email"
-								aria-label="newsletter"
-							/>
-							<button type="submit" class="button2">Submit</button>
-						</div>
-					</form>
+<head>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>If These Lands Could Talk</title>
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link
+		href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap"
+		rel="stylesheet" />
+	<link rel="stylesheet" href="./css/normalize.css" />
+	<link rel="stylesheet" href="css/global.css" />
+	<!-- linked to created home.css -->
+	<link rel="stylesheet" href="./css/home.css" />
+</head>
+
+<body>
+	<!-- header section -->
+
+	<!-- =======================
+		Hero Section
+		========================== -->
+
+	<!-- "hero" is the class for the entire section -->
+	<section class="hero">
```

Comment:
Can we add the navigation here?

Reply:
the navigation was merged in on line 20 (after original PR was sent), so this is done and resolved :)